documentation/release-notes-2-8.md

```diff
@@ -34,32 +31,43 @@ and finally a boot loader is installed.
 System administrators use the ReaR framework to set up a disaster recovery procedure
 as part of their disaster recovery policy (which complements their existing backup policy).
 
-ReaR complements backup and restore of data
-but ReaR is neither a backup software nor a backup management software.
-Data backup and restore happens via dedicated backup software which is called by ReaR.
-
-ReaR has built-in support for some 'internal' backup programs like 'tar' and 'rsync'.
-
-ReaR integrates with various external (third-party) backup software solutions, for example:
-
-* IBM Storage Protect (TSM) / Tivoli Storage Manager / IBM Spectrum Protect
-* Data Protector (DP)
-* Veritas NetBackup (NBU) / Symantec NetBackup
-* Commvault Simpana (GALAXY10) / Galaxy 10
-* Bacula (BACULA)
-* Bareos (BAREOS)
-* Rsync Backup Made Easy (RBME)
-* Duplicity/Duply (DUPLICITY)
-* Dell NetWorker (NSR) / EMC NetWorker / Legato NetWorker
-* Dell EMC Avamar (AVA) / EMC Avamar
-* SEP Sesam (SESAM)
-* Borg (BORG)
-* Rubrik Cloud Data Management (CDM)
-
-ReaR integrates with Disaster Recovery Linux Manager (DRLM) <http://drlm.org>
-
-
-# System and Software Requirements
+ReaR complements backup and restore of data with bare metal disaster recovery. ReaR can also act as local backup software,
+but ReaR is not a a backup management software. In many enterprise environmentments, data backup and restore happens via dedicated backup software which is integrated by ReaR and used to restore the data onto a recovery system.
```

Comment:
I think "restore the data onto a recovery system"
sounds confusing because at least for me
"recovery system" means the "ReaR recovery system".

I think what is actually meant is something like
"restore the data onto a recreated system"
or
"restore the data onto a recreated (target) system"
or
"restore the data onto a system on replacement hardware"

What metters for me is to make the distiction obvious between
* the "original system"
* the "ReaR recovery system"
* the "target system" or "recreated system" or "recreated target system"